index.html

```diff
@@ -4,9 +4,139 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Tables</title>
+    <link rel="stylesheet" href="stylesheet/style.css">
```

Comment:
href="./style.css"

Reply:
change according to instruction there is no similarity between the attribute and file name now